DQMOffline/Trigger/plugins/MssmHbbMonitor.cc

```diff
@@ -0,0 +1,203 @@
+   
+   const edm::TriggerResults & triggers = *(triggerResultsHandler.product());
+   
+   for ( size_t j = 0 ; j < hltConfig_.size() ; ++j )
+   {
+      if ( hltConfig_.triggerName(j).find(pathname_) == 0 )
+      {
+         if ( triggers.accept(j) ) 
+         {
+            accept = true;
+            break;
+         }
+      }
+   }
+   
+   if ( accept )
+   {
+      
+      reco::JetTagCollection jettags = *(offlineJetTagPFHandler.product());
+      if ( jettags.size() > 1 )
+      {
+         const reco::Jet * jet1 = jettags.key(0).get();
+         const reco::Jet * jet2 = jettags.key(1).get();
+         TLorentzVector p4_jet1;
+         p4_jet1.SetPtEtaPhiM(jet1->pt(),jet1->eta(),jet1->phi(),0);
+         TLorentzVector p4_jet2;
+         p4_jet2.SetPtEtaPhiM(jet2->pt(),jet2->eta(),jet2->phi(),0);
+         float btag1 = jettags.value(0);
+         float btag2 = jettags.value(1);
+         if ( btag1 < 0 ) btag1 = -0.5;
+         if ( btag2 < 0 ) btag2 = -0.5;
+         
+         float deta12 = p4_jet1.Eta() - p4_jet2.Eta();
+         float dphi12 = p4_jet1.DeltaPhi(p4_jet2);
+         float dr12   = p4_jet1.DeltaR(p4_jet2);
+            
+         if ( jet1->pt() > jetPtmin_ && jet2->pt() > jetPtmin_ && fabs(jet1->eta()) < jetEtamax_ && fabs(jet2->eta()) < jetEtamax_ )
+         {
+            pt_jet1_  -> Fill(jet1->pt());
+            pt_jet2_  -> Fill(jet2->pt());
+            eta_jet1_ -> Fill(jet1->eta());
+            eta_jet2_ -> Fill(jet2->eta());
+            phi_jet1_ -> Fill(jet1->phi());
+            phi_jet2_ -> Fill(jet2->phi());
+            deta_jet12_ -> Fill(deta12);
+            dphi_jet12_ -> Fill(dphi12);
+            dr_jet12_   -> Fill(dr12);
+            eta_phi_jet1_ -> Fill(jet1->eta(),jet1->phi());
+            eta_phi_jet2_ -> Fill(jet2->eta(),jet2->phi());
+            
+            discr_offline_btag_jet1_ -> Fill(btag1);
+            discr_offline_btag_jet2_ -> Fill(btag2);
+            
+            
+            if ( doMuon )
+            {
+               reco::MuonCollection muons = *(muonHandler.product());
+               for ( auto & muon : muons )
+               {
+                  if ( muon.pt() < muonPtmin_ || fabs(muon.eta()) > muonEtamax_ || !muon.isGlobalMuon()) continue;
+                  TLorentzVector p4_muon;
+                  p4_muon.SetPtEtaPhiM(muon.pt(),muon.eta(),muon.phi(),0);
+                  float dR1 = p4_muon.DeltaR(p4_jet1);
+                  float dR2 = p4_muon.DeltaR(p4_jet2);
+                  if ( dR1 < jetMuonDRmax_ || dR2 < jetMuonDRmax_ ) 
+                  {
+                     pt_muon_  -> Fill(muon.pt());
+                     eta_muon_ -> Fill(muon.eta());
+                     phi_muon_ -> Fill(muon.phi());
+                     break;
+                  }
+               }
+            }
+            
+         } // offline jets kinematic selection
+      } // at least two offline jets
+   } // accept trigger
+   
```

Comment:
ah ,this is an other code ;)
why don't you use the one developed by @defranchis ?